README.md

```diff
@@ -15,9 +15,27 @@ from scratch as well as keep it up-to-date over time.
 
 ### Prerequisites
 
+* `brew`
```

Comment:
needs git to clone... brew to install git or do the hipster thing and curl...see discussion below.